tools/docs-content/check-package-json-ci.js

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env node
+
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
```

Comment:
Do we need this is every script given that we have https://github.com/angular/material.angular.io/blob/master/LICENSE for the whole repo?

We should probably bump https://github.com/angular/material.angular.io/blob/master/LICENSE to 2022 as well.

Reply:
It's probably not needed but shouldn't hurt. We are doing this in all dev-infra shared scripts and in FW as well.

Yeah, seems like it. That could be a separate change I guess